packages/fromHttpRequest/tsconfig.json

```diff
@@ -2,7 +2,8 @@
   "extends": "../../tsconfig",
   "compilerOptions": {
     "rootDir": "./src",
-    "outDir": "./dist"
+    "outDir": "./dist",
+    "lib": ["es2022", "dom"]
```

Comment:
Had to do this in a few places, since dom types are being used and were previously being leaked